openedx/core/djangoapps/programs/utils.py

```diff
@@ -266,7 +266,8 @@ def reshape(course_run):
                 # count towards completion of a course in a program). This may change
                 # in the future to make use of the more rigid set of "applicable seat
                 # types" associated with each program type in the catalog.
-                'type': course_run['type'],
+                # LEARNER-1274 Course types verified are counted as credit.
```

Comment:
No. Runs of type "credit" are being counted as "verified." This should be something like:
> Runs of type 'credit' are counted as 'verified' since verified certificates are earned when credit runs are completed. LEARNER-1274 tracks a cleaner way to do this using the discovery service's applicable_seat_types field.